File: plugin/src/main/groovy/ru/cleverpumpkin/badge/PluginExtension.java
package ru.cleverpumpkin.badge;

import javax.annotation.Nullable;

public class PluginExtension {

    public final String name;

    public int fontSize = 10;
    public boolean enabled = false;
    public String textColor = "#ffffff";
    public String labelColor = "#9C000000";
    @Nullable
    public String text = null;

    public PluginExtension(String name) {
        this.name = name;
    }

}

File: plugin/src/main/java/ru/cleverpumpkin/appbadge/extension/PluginExtension.kt
package ru.cleverpumpkin.appbadge.extension

/**
 * @author Sergey Chuprin
 */
class PluginExtension(
    val name: String
) {
    var fontSize = 10
    var enabled = false
    var textColor = "#FFFFFF"
    var labelColor = "#9C000000"
    var text: String? = null
}